<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Инвентар ЛСМ Склад 12</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <!-- Inventory Tab -->
      <widget class="QWidget" name="inventory_tab">
       <attribute name="title">
        <string>Машини Инвентар</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLineEdit" name="inventory_search_input">
          <property name="placeholderText">
           <string>Търсене по име или номер...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="inventory_inputs_frame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label_1">
             <property name="text">
              <string>Инвентарен номер:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="inventory_number_input"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Сериен номер:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="serial_number_input"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Име:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="name_input"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Тип:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="type_input"/>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Стойност:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLineEdit" name="value_input"/>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Състояние:</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QLineEdit" name="condition_input"/>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Дата на придобиване:</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QDateEdit" name="date_input"/>
           </item>
           <item row="7" column="1">
            <widget class="QPushButton" name="btn_save_inventory">
             <property name="text">
              <string>Запази</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="btn_add_inventory">
            <property name="text">
             <string>Добави в Инвентара</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_delete_inventory">
            <property name="text">
             <string>Изтрий Избраното</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="inventory_table">
          <column>
           <property name="text">
            <string>ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Инв. Номер</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Сер. Номер</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Име</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Тип</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Стойност</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Състояние</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Дата</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Под наем</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="btn_prev_inventory_page">
            <property name="text">
             <string>Предишна страница</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_next_inventory_page">
            <property name="text">
             <string>Следваща страница</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      
      <!-- Rental Tab -->
      <widget class="QWidget" name="rental_tab">
       <attribute name="title">
        <string>Машини под Наем</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLineEdit" name="rental_search_input">
          <property name="placeholderText">
           <string>Търсене по име или номер...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="rental_inputs_frame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Инвентарен номер:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="rental_inventory_number_input"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Сериен номер:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="rental_serial_number_input"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Име:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="rental_name_input"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Дневен наем:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="daily_rate_input"/>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>Стойност:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLineEdit" name="rental_value_input"/>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>Дата на поддръжка:</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QDateEdit" name="maintenance_date_input"/>
           </item>
           <item row="6" column="1">
            <widget class="QPushButton" name="btn_save_rental">
             <property name="text">
              <string>Запази</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="btn_add_rental">
            <property name="text">
             <string>Добави във 'Под Наем'</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_delete_rental">
            <property name="text">
             <string>Изтрий Избраното</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="rental_table">
          <column>
           <property name="text">
            <string>ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Инв. Номер</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Сер. Номер</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Име</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Дневен Наем</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Стойност</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Дата</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="btn_prev_rental_page">
            <property name="text">
             <string>Предишна страница</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_next_rental_page">
            <property name="text">
             <string>Следваща страница</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>